netutil.py

```diff
@@ -84,6 +84,15 @@ def writeResolverFile(configuration, filename):
 
 interface_up = {}
 
+def restartSystemdNetworkdService(timeout=20):
```

Comment:
Wouldn't calling `networkctl reload` be the preferred way to do this rather than fully restarting the daemon?